### ISSUE TITLE
Feature Request/Idea: Landing pages for subcollections

### ISSUE THREAD

**Issue by sergejzr** [IQSS/dataverse#10252]
Dear Dataverse developers.
Dataverse has a possibility to define a [custom_homepage](https://guides.dataverse.org/en/latest/installation/config.html#id187) as branding for the whole Dataverse installation. This is also what we did for our [bonndata](https://bonndata.uni-bonn.de/). It shows features like overall statistics, recent and popular datasets, etc.
**Overview of the Feature Request**
Now, the admins of our subcollections would like to obtain similar landing pages for their collections. To do this, at the moment, we would need to generate a header and a footer for these collections as well.
More naturally would be to be able to add a custom branding for a subcollection in the similar way as the main landing page is added.

**What kind of user is the feature intended for?**
(Example users roles: API User, Curator, Depositor, Guest, Superuser, Sysadmin)
Superusers of the Subdataverse

**What inspired the request?**
User requests

**What existing behavior do you want changed?**
Same as with http://localhost:8080/api/admin/settings/:HomePageCustomizationFile
but for a specified collection

**Any open or closed issues related to this feature request?**
I may have overlooked something, but I did a research and could not find similar requests

**Comment by yjyale** [comment]
This feature will be very useful. Looking forward to having this feature available soon!

**Comment by pdurbin** [comment]
I noticed the discussion at https://groups.google.com/g/dataverse-community/c/8bh76NnF_m8/m/xwP2dbVGAAAJ

Yes, this would be a nice feature but we are in the process of rewriting the UI of Dataverse in React and are trying to touch the old frontend code (in this repo) as little a possible. Please feel free to open an issue at https://github.com/IQSS/dataverse-frontend

**Comment by yjyale** [comment]
That'll be great! Will the new UI replace the old UI for Dataverse community? Will the new UI allow customized branding for the differences among main header and sub-collections' headers?

**Comment by pdurbin** [comment]
@yjyale yes, the new UI will someday replace the old UI. It will be a long process but you can watch these videos to see the progress: https://github.com/IQSS/dataverse-frontend#demo-videos

The new UI will not allow customized branding for the differences among main header and sub-collections' headers. However, you are welcome to create an issue in that repo.

Basically, the strategy with developing the new UI is to replicate what is in the old UI. So because the feature you want isn't in the old UI, it will not be implemented in the new UI unless there's an issue that is prioritized.

I hope this helps!

**Comment by cmbz** [comment]
2025-06-25
- Closing
- @sergejzr If you are still interested in this features, please create an issue in the frontend repo, as per Phil's suggestion here: https://github.com/IQSS/dataverse/issues/10252#issuecomment-2047853713
